docs/Developers/Newcomers_Guide.rst

```diff
@@ -442,7 +442,7 @@ Step 5. Creating a Pull Request
 
 Now that your commit has been sent to your fork, it is time
 to create a ``Pull Request``. You can do this by accessing your fork on GitHub
-and clicking ``New Pull Request``.
+and clicking ``New Pull Request``. After you submit your pull request, **DO NOT click the 'Update Branch' button**. When asked for a rebase, consult <http://coala.io/rebase> instead.
```

Comment:
This line is a bit too long, please break it down into multiple lines.

Reply:
I wonder if it's even needed in the first place!?